src/Components/test/testassets/ComponentsApp.Server/Pages/Index.cshtml

```diff
@@ -1,19 +1,42 @@
-@page "{*clientroutes}"
+@page
```

Comment:
This seems a bit complicated and test-specific. Would it be worth making this an equivalent to https://github.com/aspnet/AspNetCore/pull/8888/files#diff-2dc5f835023a5c0887aa867d4c20961a once we merge https://github.com/aspnet/AspNetCore/pull/8888?

Then we won't need any of the querystring matching logic, plus the E2E test will have the option of making different assertions before/after the transition into interactive mode.

Reply:
I'll update this to your version. This is what I conjured yesterday at 10pm :smile:

Reply:
I updated this to simply add a parameter when we prerender the app and make sure its there when we reconnect